lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -1,3 +1,6 @@
+@use '@hypothesis/frontend-shared/styles/mixins/layout';
+@use '@hypothesis/frontend-shared/styles/components/buttons';
```

Comment:
Is this used?

Reply:
left over vestige from previous effort, good catch.